proto/process/agent.proto

```diff
@@ -77,9 +77,14 @@ message CollectorConnections {
 	// Mapping of processes running in each container
 	map<int32, string> containerForPid = 10;
 
+	// encoded bytes here are use to store a tags []string
+	// the tags encoder is more space efficient than protobuf
 	bytes encodedTags = 11;
```

Comment:
What is this field? I don't see it being used in the agent.

Reply:
it's used by the backend only @kevinconaway https://github.com/DataDog/agent-payload/pull/120#issuecomment-921828754